electron/SatelliteApp.ts

```diff
@@ -252,8 +252,7 @@ export class SatelliteApp {
                 PM2_API_PORT: this.satelliteSettings.pm2Port,
                 // PM2_HOME: this.pm2_home,
```

Comment:
maybe PM2 needs proxy too? it reports to home a cool staff of data

Reply:
I would prefer to add it later when needed, but it's up to you